client/views/app/flexTabBar.coffee

```diff
@@ -2,6 +2,7 @@ Template.flexTabBar.helpers
 	active: ->
 		return 'active' if @template is RocketChat.TabBar.getTemplate() and RocketChat.TabBar.isFlexOpen()
 	buttons: ->
+		RocketChat.TabBar.getButtons()
```

Comment:
@marceloschmidt yeah that for sure shouldn't be there.  It'll make that be called twice, which would be pretty pointless